organization-bootstrap/environments/main.tf

```diff
@@ -0,0 +1,159 @@
+# Copyright 2019 Google LLC
```

Comment:
I'd encourage decoupling this file into projects.tf, folders.tf, export.tf, etc.

Reply:
Thanks, but especially for simple environments like this, I much prefer having everything together to allow easy parsing.